registrations/tests/test_seatsreservation_post.py

```diff
@@ -56,6 +57,8 @@ def test_seats_value_is_required(api_client, event, registration):
 def test_cannot_reserve_seats_if_there_are_not_enough_seats_available(
     api_client, event, registration
 ):
+    activate("en")
```

Comment:
This is used in so many tests that it probably makes sense to make it the default in all registrations' tests. That can be achieved by creating an automatically used fixture in `registrations/tests/conftest.py`
```python
@pytest.fixture(autouse=True)
def use_english():
    with translation.override("en"):
        yield
```

Reply:
Changed as suggested.